content/md/imports-exports/monitor-jobs.md

```diff
@@ -9,50 +9,62 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
+![List of jobs in process tracker](../img/Jobs_process-tracker.png)  
+Note that you can also check the status of your launched jobs on the dashboard. To access it, go to `Activity` and click on `Dashboard`.
+![List of jobs in dashboard](../img/Jobs_dashboard.png)
+1.  Finally, when the job is complete, a notification appears at the top of your screen. You can always go back to the job itself if you need more details.
+![Notification of a job complete](../img/Jobs_notification.png)
 
 ::: tips
-If you want to view the jobs of all the users of your PIM (and not only yours), adding the [permission on action](manage-the-interface-and-actions-accesses.html#rights-on-system-pages) `View all jobs in Process Tracker` on your user role will grant you that privilege.
+Please note it is possible to use the `View all jobs in Process Tracker` permission in the `Roles` page to access all the jobs launched by all the users of your PIM. To find out more, check our article on [permissions](manage-the-interface-and-actions-accesses.html#rights-on-system-pages) in our help center.
 :::
 
-## Imports & exports reports
+# What can you find in a job detail?
+
+A job detail is a report of a specific job.
+In each job detail, you'll find information about the status of your job, which may vary depending on their status: `Started`, `Completed`, `Failed` or `Stopped` with a specific color code. You will see:
+* The number of steps
+* A time estimation above each step
+* A table with the details of all the actions performed under each step
 
-You can also have more details when you executed an import or an export. To review their report:
-1.  Go to `Imports`
-2.  Select in the list the profile job for which you want to view the report
-2.  You can clearly see the status of your import: `Completed`, `Started` or `Failed` thanks to a color code
-1.  Click on your import/export
-1.  The page of the import/export job detail should appear
+To access a specific job detail:
+1.  Go to `Activity`/`Process Tracker`
+1.  Search for the job you would like to know more about
+1.  Click on the job
+![Job detail example](../img/Jobs_job-detail.png)
 
-![image](../img/Exports_Calculation5.png)
+# How to monitor your jobs?
+## Import & export jobs
 
-The details of the executing import/export job are shown on the page. The details are related to each executed import/export profile.
+You can get more details when you execute an import or an export job. To access the job detail:
+1.  Go to `Imports` or `Exports`
+1.  In the list, select the profile job you need
+1.  Click on your import/export job
+1.  The import/export job detail appear
+![Jobs export/import profiles](../img/Jobs_export-import-profiles.png)
 
-## Mass actions execution reports
+The details of the executed import/export job are displayed and related to individual import/export profiles.
```

Comment:
As a info note, you can also add that those jobs are also visible in the Process tracker page.